src/Dialect/ONNX/ONNXOps/OpHelper.hpp

```diff
@@ -4,15 +4,16 @@
 
 //===------- ONNXOpsHelper.hpp - Helper functions for ONNX dialects -------===//
```

Comment:
This is a nit, but the prolog "ONNXOpsHelper.hpp" does not match the file name "OpHelper.hpp"

Reply:
Thanks I agree, that was there prior. Some files need to be cleaned up.